src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiItemRecipe.java

```diff
@@ -336,6 +337,14 @@ public void handleMouseInput() throws IOException {
 		int mouseY = scaledResolution.getScaledHeight() -
 			Mouse.getY() * scaledResolution.getScaledHeight() / Minecraft.getMinecraft().displayHeight - 1;
 		getCurrentRecipe().genericMouseInput(mouseX, mouseY);
+
+		// Allow Paging with Scroll-Wheel
+		if (NotEnoughUpdates.INSTANCE.config.tooltipTweaks.scrollableTooltips &&
```

Comment:
Most people dont use neu for their tooltip scrolling, I would remove this check

Reply:
But what if people do?

Reply:
it will still block it?

Reply:
This looks to me like it *only* blocks if scrollableTooltips are turned on *and* there was a tooltip rendered last tick. I also could not reproduce the bug of it not scrolling with nothing hovered or the setting turned off.

Reply:
That's what it does! It blocks scrolling if scrolling is enabled and a tooltip is displayed. The comment was misleading, i meant it for the whole section

Reply:
no, I meant that most people use sk1ers scrollable tooltips so we shouldn't check if our scrollable tooltips are enabled because that doesn't affect 95% of users